scripts/variable_stamps.sh

```diff
@@ -0,0 +1,12 @@
+#!/usr/bin/env bash
+
+SCRIPT_DIR="$(cd "$(dirname "$0")" && pwd)"
+source "${SCRIPT_DIR}/utils.sh"
+
+set -euo pipefail
+
+gitroot="$(git rev-parse --show-toplevel)"
```

Comment:
Isn't the location of the git root determined relative to `SCRIPT_DIR` anyway? Or are you future proofing it to be used in other repos?

Reply:
Yeah, it just seems like a better option since the location of `get-tag` relative to `SCRIPT_DIR` is not constrained.